hardware/i2c.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+order:
+title: i2c
+heading: Schematics
+description: micro:bit i2c pins
+permalink: /hardware/schematic/
+ref: i2c
+lang: en
+assigned-to: markw
+review-with: jonnya
+---
+
+If you make an accessory for the micro:bit, please help us by editing the page and sharing the details of the i2c addresses you use.
+
+## Use of the shared I2C bus
+
+The motion sensors on the board are on the same I2C bus as the edge connector I2C pins. This means that if you have an accessory that uses I2C on this bus, you need to check it won’t clash with the new sensors.
```

Comment:
Technically I think people need to check it won't clash with *any* of the sensors (so that sentence is right for someone who already has an accessory, but given it's now for anyone, including people making new accessories it should become:
> The motion sensors on the board are on the same I2C bus as the edge connector I2C pins. This means that if you have an accessory that uses I2C on this bus, you need to check it won’t clash with any of the possible on-board sensors.